src/events.ts

```diff
@@ -554,3 +554,83 @@ export interface VideoPlaybackQualityChangedEvent extends Event {
    */
   oldVideoQuality: VideoQuality;
 }
+
+/**
+ * Emitted when casting to a cast-compatible device is available.
+ */
+export interface CastAvailableEvent extends Event {}
+
+/**
+ * Emitted when the playback on a cast-compatible device was paused.
+ *
+ * On Android `PausedEvent` is also emitted while casting.
+ */
+export interface CastPaused extends Event {}
```

Comment:
```suggestion
export interface CastPausedEvent extends Event {}
```
For consistency with existing event naming. Holds for all events below.

Reply:
Oops, I slipped on the multi-cursor edit.
Fixed in 56ee8d3